code/game/objects/items/kitchen.dm

```diff
@@ -153,6 +153,17 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_price = 600
 
+/obj/item/kitchen/knife/hunting
+	name = "hunting knife"
+	desc = "Despite its name, it's mainly used for cutting meat from dead prey rather than actual hunting."
+	item_state = "huntingknife"
+	icon_state = "huntingknife"
+
+/obj/item/kitchen/knife/hunting/Initialize()
+	. = ..()
+	var/datum/component/butchering/butcher = GetComponent(/datum/component/butchering)
+	butcher.bonus_modifier += 20

```

Comment:
Could you handle this in `ComponentInitialize()`?
It would be good if it was within a redefined proc there, as well, instead of using `GetComponent`